i18n/states.php

```diff
@@ -172,6 +172,20 @@
 	),
 	'BH' => array(),
 	'BI' => array(),
+	'BJ' => array( // Benin states.
+		'BJ-01' => __( 'Alibori', 'woocommerce' ),
+		'BJ-02' => __( 'Atacora', 'woocommerce' ),
```

Comment:
Noting for second reviewer that this is the French spelling for the state.

Reply:
Yes, you got a point. This is the French spelling for this state because it is a French state.